README.md

```diff
@@ -84,3 +84,15 @@ on initial render. Receives two arguments: `keypath` and `value`. Defaults to
 Optional parameters for filterer (search). Must be an object.
 
 - `ignoreCase`, Set to `true` to enable case insensitivity in search. Defaults to `false`.
+
+#### props.rootExpanded
+
+Optional boolean to expand the root object. Defaults to true.
```

Comment:
Hi, 

That sounds great. It didn't occur to me to update the existing `isExpanded` predicate.